packages/entities/src/lib/update.mutation.spec.ts

```diff
@@ -164,23 +164,10 @@ describe('update', () => {
       const todo = createTodo(1);
       store.update(addEntities([todo]));
 
-      // update the todo
```

Comment:
why removed?

Reply:
The next two lines [were](https://github.com/ngneat/elf/blob/master/packages/entities/src/lib/update.mutation.spec.ts#L168-L169):
```ts
todo.completed = true;
store.update(upsertEntities([todo]));
```
Even if you would comment out the `store.update()` line, the test would still pass since we're mutating the todo directly in the store. After changing it to the immutable version I noticed that this test became exactly the same as the next one.